### ISSUE TITLE
Force unique names for imported classic menus in Nav block

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#39200]
Currently you can import a classic menu with the Navigation block and it will create a block based Navigation with the exact same name.

The issue is that you can do this repeatedly, generating the exact same name over and over again.

Let's refactor `use-generate-default-navigation-title` to accept a title to dedupe against rather than relying on only a default:

https://github.com/WordPress/gutenberg/blob/f7e886edccee02d248e2361a23e49457b2a3cb68/packages/block-library/src/navigation/edit/use-generate-default-navigation-title.js#L27

Once this is in place the classic menu conversion process will need to pass in the name of the newly created menu thereby ensuring a unique name each time the menu is imported.

**Comment by kathrynwp** [comment]
Heya @getdave - just checking in to see if this is still on the radar? Any chance to get it into 6.1, or has that window closed? Thanks!

**Comment by getdave** [comment]
> Heya @getdave - just checking in to see if this is still on the radar?

It wasn't on our immediate radar. What level of impact do you see this having on users? Is it severe or a lower priority item?

> Any chance to get it into 6.1, or has that window closed? Thanks!

No chance as the feature freeze has happened and we're nearly at RC. Sorry!

**Comment by kathrynwp** [comment]
@getdave Probably lower priority!
